contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -993,8 +1000,11 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
     }
 
     // resolve gas limit and validate chainFamilySelector
-    if (destChainConfig.chainFamilySelector == Internal.CHAIN_FAMILY_SELECTOR_EVM) {
-      gasLimit = _parseEVMExtraArgsFromBytes(
+    if (
+      destChainConfig.chainFamilySelector == Internal.CHAIN_FAMILY_SELECTOR_EVM
+        || destChainConfig.chainFamilySelector == Internal.CHAIN_FAMILY_SELECTOR_APTOS
```

Comment:
Should we have something in `Internal` that is the source of truth of "generic" extra args chains?